src/Utilities/UnitTests/ToolLocationHelper_Tests.cs

```diff
@@ -593,9 +593,7 @@ public void TestGetPathToBuildToolsFile()
             Assert.Equal(tv12path, ToolLocationHelper.GetPathToBuildToolsFile("MSBuild.exe", ToolLocationHelper.CurrentToolsVersion));
         }
 
-        [Fact(Skip = "Ignored in MSTest")]
-
-        // Ignore: Test requires installed toolset.
+        [Fact(Skip = "https://github.com/Microsoft/msbuild/issues/722")]
```

Comment:
Added the ignore in a separate commit so it's easily revertable once #722 is understood.